### PR TITLE
10.0

### PR DESCRIPTION
**Pull request by antonyrajserpentcs**
Hello Sir,
I updated changes which were given.
Please Verify it.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,15 +2,15 @@
...
 Odoo Connector WooCommerce Modules
-===================================
+==================================
 *Odoo WooCommerce Connector (also known as connector_woocommerce) is a bi-directional connector, 100% compatible with the last OpenERP 10.0 and latest WooCommerce versions (2.4.x).*
 
 [//]: # (addons)
 
-Unported addons
+Available addons
 ---------------
```
Comment (JayVora-SerpentCS) [inline README.md]: - to add

File: connector_woocommerce/__manifest__.py [hunk 2]
```diff
@@ -26,7 +26,7 @@
     'description': """WooCommerce Connector.""",
     'author': 'Tech Receptives',
     'contributors': 'Tech Receptives, Serpent Consulting Services Pvt. Ltd.',
-    'maintainer': 'OCA',
+    'maintainer': 'Odoo Community Association',
```
Comment (JayVora-SerpentCS) [inline connector_woocommerce/__manifest__.py]: Odoo Community Association (OCA)

File: connector_woocommerce/components/backend_adapter.py [hunk 3]
```diff
@@ -35,18 +35,19 @@
 try:
     from woocommerce import API
 except ImportError:
-    _logger.debug("Cannot import 'woocommerce'")
+    _logger.debug("Can not import 'woocommerce'")
```
Comment (JayVora-SerpentCS) [inline connector_woocommerce/components/backend_adapter.py]: Cannot is right.

File: connector_woocommerce/model/woo_binding/common.py [hunk 4]
```diff
@@ -51,10 +51,8 @@ class WooBinding(models.AbstractModel):
 
     @job(default_channel='root.woo')
     @api.model
-    def import_batch(self, backend, filters=None):
+    def import_batch(self, backend, filters=dict()):
```
Comment (JayVora-SerpentCS) [inline connector_woocommerce/model/woo_binding/common.py]: any logic behind replacing None by dict()?

Comment (antonyrajserpentcs) [reply connector_woocommerce/model/woo_binding/common.py]: Yes, Before when the filters was none, it was checked with if condition and set as empty dictionary. As of now if condition is removed, so made the filters as empty dictionary by default.
Before,
def import_batch(self, backend, filters=None):
        """ Prepare the import of records modified on Woo """
        if filters is None:
            filters = {}
After,
def import_batch(self, backend, filters=dict()):
        """ Prepare the import of records modified on Woo """

**Pull request merged**